plugins/analysis-nori/src/main/java/org/elasticsearch/plugin/analysis/nori/NoriTokenizerFactory.java

```diff
@@ -80,7 +80,7 @@ public static UserDictionary getUserDictionary(Environment env, Settings setting
      * that the duplicate check feature is supported.
      */
     private static boolean isSupportDuplicateCheck(IndexSettings indexSettings) {
-        return indexSettings.getIndexVersionCreated().onOrAfter(IndexVersions.ES_VERSION_8_13);
+        return indexSettings.getIndexVersionCreated().onOrAfter(IndexVersions.ES_VERSION_8_13); // HERE
```

Comment:
This will be removed. It is highlighted here to show the usage of `ES_VERSION_8_13`, since I'm proposing to reuse that name with a new id.

Reply:
So, this is potentially tricky. This functionally changes the check from 8500007 to 8500009. Is Nori used in serverless at all?

Reply:
Good point. Let's assume that it is used. I'll update the constant name to include NORI, and also update the use-sites so that it uses the new constant (and therefore the same id as it previous did)

Reply:
Hmm..  now I'm stuck. If you read the method javadoc description on this method, `isSupportDuplicateCheck`, then  it wants to know if the version of ES is greater than 8.12, but we cannot change the constant id, since it could be used in _main_.

Reply:
I believe we'll have to put a couple of excludes in this conditional check, right?  e.g. on or after the the existing `id`, but not the 9_9_1 upgrade. Lemme try to write the code for that.

Reply:
It depends what else needs this check - is there other functionality in 8.13 that this requires? (I'm not sure an indexversion is the right sort of thing to use here, but here we are...)